doc/whats_new/v1.2.rst

```diff
@@ -301,6 +301,10 @@ Changelog
 
 - |Feature| Adds `class_weight` to :class:`ensemble.HistGradientBoostingClassifier`.
   :pr:`22014` by `Thomas Fan`_.
+  
+- |Feature| Add `oob_scores_` to class:`ensemble.GradientBoostingClassifier` to
+  store OOB Loss.
+  :pr:`24882` by :user:`Ashwin Mathur <awinml>`.
```

Comment:
Now that 1.2 is on track to be release, this entry needs to be moved to `doc/whats_new/v1.3.rst`.